learningReact/demo_react/src/Props/DemoProps.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+
+
+class DemoProps extends Component{
```

Comment:
Let double check `stateless` and `stateful` as I mention before. Is this case,  use stateless instead.